kyuubi-common/src/main/scala/org/apache/kyuubi/engine/deploy/yarn/EngineYarnModeSubmitter.scala

```diff
@@ -17,10 +17,12 @@
 package org.apache.kyuubi.engine.deploy.yarn
```

Comment:
The  submitter process determine whether to login as well in kerberized environment?

Reply:
I'm not sure what "determine whether to login as well" means. Can you elaborate it?